src/quickgui/plugin/qgsquickmessagelog.qml

```diff
@@ -0,0 +1,71 @@
+/***************************************************************************
+ qgsquickmessagelog.qml
+  --------------------------------------
+  Date                 : January 2018
+  Copyright            : (C) 2018 by Peter Petrik
+  Email                : zilolv at gmail dot com
+ ***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+import QtQuick.Controls 2.0
+import QtQuick 2.5
+import QgsQuick 0.1 as QgsQuick
+
+Item {
+  property alias model: table.model
```

Comment:
document properties (some simple comments what it does if unclear)
see map canvase.qml for reference

also some simple description what this component does (something like shows all messages from qgsmessagelog ,,,)